ServerProject-DONT-IMPORT-INTO-UNITY/LRM/RelayHandler/RelayHandler.cs

```diff
@@ -21,15 +21,27 @@ public RelayHandler(int maxPacketSize)
         private bool DoesServerIdExist(string id) => _cachedRooms.ContainsKey(id);
 
         private string GenerateRoomID()
+        {
+            return GenerateRoomID(false);
+        }
+            
+        private string GenerateRoomID(bool generateNumerical)
```

Comment:
Instead of accepting a bool flag to toggle functionality it would be more flexible to accept either a new string or char[] argument as a set of values to use. Numerical would just be accepting "0123456789", and then accepting other combinations such as alphanumeric, base64, hex, or whatever people want to use is just a matter of passing in the correct arg. It also keeps all the logic inside the method the same. You could also accept a length variable as well but that is probably outside the scope of this change.

You could also make the string/char[] parameter optional and default to "ABCDEFGHIJKLMNOPQRSTUVWXYZ" instead of needing to overload.

Reply:
awesome suggestion